src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -111,6 +130,24 @@ def repo_status_icon(status, details = nil)
     sprite_tag icon, title: description
   end
 
+  def repo_status_icon_webui2(status, details = nil)
```

Comment:
I would rather have a partial then this helper. It would be more readable and it's only called once.

Reply:
:thinking:

Reply:
I prefer the helper method though